org.telegram.desktop.json

```diff
@@ -38,6 +38,14 @@
     "cleanup-commands": [
         "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
     ],
+    "build-options": {
+        "cflags": "",
+        "cflags-override": true,
+        "cxxflags": "",
+        "cxxflags-override": true,
+        "ldflags": "",
```

Comment:
No need to override ldflags, the defaults are `-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed` (and the first one is likely to be essential).

Reply:
I want to replicate official build environment, where only flags from tdesktop cmake files are set.

Reply:
or instance, relro is a hardening flag AFAIR, i don't think it is a good idea to have it given that tdesktop doesn't support building in hardening environments.